gradoop-flink/src/main/java/org/gradoop/flink/model/impl/functions/epgm/EdgeFromIds.java

```diff
@@ -57,9 +58,6 @@ public EdgeFromIds(EPGMEdgeFactory<Edge> epgmEdgeFactory) {
   }
 
   /**
-   * Initializes an {@link Edge} from a given {@link GradoopId} triple. The
```

Comment:
You can remove this comment entirely, since this method is declared by an interface where it should already be documented.